shared/profile/wot-author/index.tsx

```diff
@@ -188,6 +194,79 @@ export const Question2Wrapper = (
   )
 }
 
+export const ReviewWrapper = (
+  props: Container.RouteProps<{
+    sigID: string // sigID of the vouch.
+  }>
+) => {
+  const sigID = Container.getRouteProps(props, 'sigID', '')
+  const dispatch = Container.useDispatch()
+  const voucheeUsername = Container.useSelector(state => state.config.username)
+  const details = Container.useSelector(state => Tracker2Constants.getDetails(state, voucheeUsername))
+  const wotEntry = (details.webOfTrustEntries || []).find(entry => entry.proofID === sigID)
+  const error = Container.useSelector(state => state.profile.wotAuthorError)
+  const [lastAction, setLastAction] = React.useState<ReviewAction | undefined>(undefined)
+  const isWaiting = Container.useAnyWaiting(UsersConstants.wotReactWaitingKey)
+  let waiting = isWaiting ? lastAction : undefined
+  let fatalError = ''
+  if (!sigID) {
+    fatalError = 'Routing missing ID.'
+  }
+  if (!wotEntry) {
+    fatalError = 'Statement not found.'
+  }
+  if (fatalError || !wotEntry) {
+    return (
+      <Review
+        disabled={true}
+        error={fatalError}
+        firstDraft={false}
+        onAccept={() => {}}
```

Comment:
better to make these undefined or functions instead of passing empty functions. same for the array below